remoteworkitem/remoteworkitem.go

```diff
@@ -54,10 +54,10 @@ const (
 	remoteDescription         = workitem.SystemDescription
 	remoteState               = workitem.SystemState
 	remoteItemID              = workitem.SystemRemoteItemID
-	remoteCreatorLogin        = "system.creator.login"
-	remoteCreatorProfileURL   = "system.creator.profile_url"
-	RemoteAssigneeLogins      = "system.assignees.login"
-	RemoteAssigneeProfileURLs = "system.assignees.profile_url"
+	remoteCreatorLogin        = "system_creator_login"
+	remoteCreatorProfileURL   = "system_creator_profile_url"
```

Comment:
As discussed on a call last Friday, please beware that we cannot simply replace `system.` with `system_` but that we also might have to deal with multiple JSONAPI violations per field name.

Reply:
@kwk Yes, I've created a static map that would deal with mapping the fields for now https://github.com/fabric8-services/fabric8-wit/pull/2333/commits/0433e16f4d577eceae05db3758ec11601d785698